Tribler/Test/test_my_channel.py

```diff
@@ -31,7 +32,7 @@ def added_rss():
                 60, lambda: len(managefiles.GetItems()) > 0, do_files_check, 'Channel did not have torrents')
 
         def do_rss():
-            self.managechannel.rss_url.SetValue(r'http://torrent.fedoraproject.org/rss20.xml')
+            self.managechannel.rss_url.SetValue(os.path.join(TESTS_DATA_DIR, 'test_rss.xml'))
```

Comment:
maybe we should test both local and remote URIS? I guess we could create a local http server that serves that same xml?